### DISCUSSION TITLE
Quick Support Version

### DISCUSSION THREAD

**Discussion by zazazouthecat** [rustdesk/rustdesk#1090] [Q&A] [answered]
Hello,
it's possible to have a UI version with just ID and PASSWORD ?
I would like to deploy it in my company but I want users to have a simplified version and that there is no possibility of connecting to a colleague.

the version with the possibility of connection would be only for the technicians of the support

**Comment by stirep** [comment]
This would be awesome!

**Reply by answer22oven** [reply]
+1, we keep using teamviewer quicksupport exclusively for this!

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/8981#discussioncomment-10263847
